compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/SkiaColorFilter.skiko.kt

```diff
@@ -33,12 +33,23 @@ fun org.jetbrains.skia.ColorFilter.asComposeColorFilter(): ColorFilter = ColorFi
 internal actual fun actualTintColorFilter(color: Color, blendMode: BlendMode): ColorFilter =
     ColorFilter(SkiaColorFilter.makeBlend(color.toArgb(), blendMode.toSkia()))
 
-internal actual fun actualColorMatrixColorFilter(colorMatrix: ColorMatrix): ColorFilter =
-    ColorFilter(
+internal actual fun actualColorMatrixColorFilter(colorMatrix: ColorMatrix): ColorFilter {
+    // as per https://developer.android.com/reference/kotlin/androidx/compose/ui/graphics/ColorMatrix
+    // columns 0, 1, 2, 3 are raw coefficients and 4th column is offset in 0..255 range
+    // skia expects offset in 0..1 range (https://fiddle.skia.org/c/2b8d12a11e9d22b0d1b85e5b0179cd4b)
```

Comment:
fiddle example doesn't look like valid proof

Reply:
Android does it here - https://cs.android.com/android/platform/superproject/+/master:frameworks/base/libs/hwui/jni/ColorFilter.cpp;l=46-54

Reply:
I'm fine with unrolling loop, but multiply on (1.0 / 255.0) - why?

Reply:
> you can just type in the numbers in fiddle and see the result

Let's put official docs/spec instead. Or link to actual android implementation at least.

>  but multiply on (1.0 / 255.0) - why?

I had two reasons here: repeat original implementation and flashbacks from low-level C world. Since here we're inside managed env, I don't think that it's really matter. The old rule in my head is "multiplication usually faster than division, never slower". Again, it's not really matter here, not a blocker at all.

Reply:
The ColorMatrix documentation in code itself describes the expected values:
https://github.com/JetBrains/compose-multiplatform-core/blob/f516176c383760c9b445d22f7f91098496020bbd/compose/ui/ui-graphics/src/commonMain/kotlin/androidx/compose/ui/graphics/ColorMatrix.kt#L24-L62